replica/replica.go

```diff
@@ -213,6 +218,29 @@ func (r *Replica) diskPath(name string) string {
 	return path.Join(r.dir, name)
 }
 
+func (r *Replica) isExtentSupported() error {
+	filePath := r.diskPath(r.info.Head + metadataSuffix)
+	fileInfo, err := os.Stat(filePath)
+	if err != nil {
+		return err
+	}
+
+	file, err := os.OpenFile(filePath, os.O_RDONLY, 0666)
+	if err != nil {
+		if os.IsPermission(err) {
+			logrus.Errorf("failed to read %s, error: Read permission denied", filePath)
```

Comment:
Don't need to single this out? `err` should contain the message already.

Reply:
roger that